x-pack/legacy/plugins/siem/public/pages/case/components/case_header_page/index.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { HeaderPage, HeaderPageProps } from '../../../../components/header_page';
+import * as i18n from './translations';
+
+const CaseHeaderPageComponent: React.FC<HeaderPageProps> = props => <HeaderPage {...props} />;
+
+CaseHeaderPageComponent.defaultProps = {
```

Comment:
what is the purpose of make this its own component?

Reply:
The only reason to have an extra component is to have defaultProps for all pages. We can add or remove the badge option to all pages at once, like the Beta badge we are using at the moment. If a specific page wants to change the badge it can change the badgeOptions.

Reply:
We can also have a default `backOptions`